src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml

```diff
@@ -74,6 +74,10 @@
             <tag name="form.type" />
             <argument type="service" id="translator" on-invalid="ignore" />
         </service>
+        <service id="form.type.color" class="Symfony\Component\Form\Extension\Core\Type\ColorType" public="true">
```

Comment:
The service doesn't need to be public, does it?

Reply:
Indeed, that was just a too quick copy paste.